notebooks/WET/WET-007/WET-007_20241028_tie-vs-danish.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>In the beginning it would be also good to mention explicitly that it was run on the "Summit LSP"  (not USDF RSP). May help with replicating the results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
This was actually run on USDF. (you can tell because the butler is the embargo repo) I added a note on this to the top.